plugins/nginx-vhosts/install

```diff
@@ -53,10 +53,10 @@ gpasswd -M "" dokku
 [[ -f /etc/logrotate.d/nginx ]] && sed -i -e 's/create 0640 www-data dokku/create 0640 www-data adm/g' /etc/logrotate.d/nginx
 
 # Create nginx error templates
-sudo mkdir -p /etc/nginx/dokku-errors
-sudo cp ./templates/400-error.html /etc/nginx/dokku-errors/400-error.html
-sudo cp ./templates/404-error.html /etc/nginx/dokku-errors/404-error.html
-sudo cp ./templates/500-error.html /etc/nginx/dokku-errors/500-error.html
+sudo mkdir -p $DOKKU_LIB_ROOT/data/nginx-vhosts/dokku-errors
+sudo cp $PWD/templates/400-error.html $DOKKU_LIB_ROOT/data/nginx-vhosts/dokku-errors/400-error.html
```

Comment:
You need to quote variable usage. Something like:

`"${PWD}/templates/400-error.html"`

Otherwise it'll fail shellcheck and tests won't pass :(